recipes/confu_json/all/conandata.yml

```diff
@@ -2,3 +2,18 @@ sources:
   0.0.5:
     url: https://github.com/werto87/confu_json/archive/refs/tags/v0.0.5.tar.gz
     sha256: bc506d4b7836a7689b1c6a2d89bb0c3441f774c8f845fef79d85c71099df5bf9
+  0.0.6:
```

Comment:
please, add only the latest version and what is really relevant to your project. Avoid adding outdated versions.